prov/rxm/src/rxm_cq.c

```diff
@@ -1870,6 +1877,60 @@ void rxm_thru_comp_error(struct rxm_ep *ep)
 	}
 }
 
+/*
+ * Right now the peer CQ is only used for collectives. There are two cases:
+ * (1) collective operations initiated by rxm and completed by util_coll or
```

Comment:
I'd say app here instead of rxm, basically distinguish between app initiated versus internal transfers.

Reply:
Good point.